functions/openhab.bash

```diff
@@ -182,13 +189,29 @@ multitail_openhab_scheme() {
   if cond_redirect sed -i -e 's|^# misc.*$|# openHAB logs\\ninclude:/etc/multitail.openhab.conf\\n#\\n# misc|g' /etc/multitail.conf; then echo "OK"; else echo "FAILED (include)"; return 1; fi
 }
 
+## Function to check if openHAB 2 is installed on the current system. Returns
+## 0 / true if openHAB is installed and 1 / false if not.
+##
+##    openhab2_is_installed()
+##
+openhab2_is_installed() {
+  if [[ $(dpkg -s 'openhab2' | grep Status | cut -d' ' -f2) == "install" ]]; then return 0; else return 1; fi
```

Comment:
use the `dpkg -s 'openhab2' &> /dev/null` style to avoid unwanted output on the user side.

Reply:
ok